android/src/main/java/com/opentokreactnative/OTSessionManager.java

```diff
@@ -247,7 +247,7 @@ public void run() {
                 }
                 mSubscriberViewContainers.remove(mStreamId);
                 if (mSubscriber != null) {
-                    mSubscriber.destroy();
+                    // mSubscriber.destroy(); From 2.17.0, resources will be automatically released by the garbage collector.
```

Comment:
I think we can just remove this if block entirely.

Reply:
I agree, maybe just note this in the changelog.